### ISSUE TITLE
% present on cohort show page is incorrect

### ISSUE THREAD

**Issue by jshawl** [ga-dc/garnet#552]
shows 100% despite 6 tardies

**Comment by RobertAKARobin** [comment]
It's tardy plus present. It was changed in one of my recent pull requests.

**Comment by jshawl** [comment]
os so % present is actually % present - % absent

**Comment by RobertAKARobin** [comment]
No, the "present" status is more accurately "on time". %Present is on time plus tardy, or absent minus tardy. Wonder if we should rename that status?

**Comment by jshawl** [comment]
![](https://dl.dropbox.com/s/nwz9djfcil4pgfj/Screenshot%202016-03-14%2013.15.14.png?dl=0)

**Comment by jshawl** [comment]
maybe just pull % present column and replace with # tardy

**Comment by RobertAKARobin** [comment]
Rather we leave it in because of the 80% present policy.

**Comment by jshawl** [comment]
https://github.com/ga-dc/wdi8/blame/f48bfc2ea9fa61477d689b700678714ad16930b5/incentives.md#L25

**Comment by RobertAKARobin** [comment]
Son of a hitch.

**Comment by jshawl** [comment]
also 80% present would be comical - that's missing one class a week - 12 absences!!!
